### ISSUE TITLE
Build arm64 docker image get error

### ISSUE THREAD

**Issue by xinatcg** [heroiclabs/nakama#1138]
Before you submit an issue we recommend you drop into the [community forum](https://forum.heroiclabs.com) and ask any questions you have or mention any problems you've had getting started with the server.

Please provide as much information as you can with this issue report. If you believe it may be an issue with one of the client libraries please report it on their [own trackers](https://github.com/heroiclabs?utf8=%E2%9C%93&q=nakama%20AND%20sdk&type=&language=).

## Description

Build arm64 docker image get error

```
docker build "$PWD" --platform "linux/arm64" --file ./Dockerfile.arm64 --build-arg commit="$(git rev-parse --short HEAD 2>/dev/null)" --build-arg version=3.19.0 -t local/nakama:3.19.0
```
 
 error:

```
8.692 gcc: error: unrecognized command-line option '-m64'
```
full

```
 => ERROR [builder 4/4] RUN git checkout --quiet "7d5051fb" &&     go build -o /go/build-out/nakama -trimpath -mod=vendor -gcflags "-trimpath $PWD" -asmflags "-trimpath $PWD" -ldflags "-s -w -X   34.1s
------
 > [builder 4/4] RUN git checkout --quiet "7d5051fb" &&     go build -o /go/build-out/nakama -trimpath -mod=vendor -gcflags "-trimpath $PWD" -asmflags "-trimpath $PWD" -ldflags "-s -w -X main.version=3.19.0 -X main.commitID=7d5051fb":
8.692 # runtime/cgo
8.692 gcc: error: unrecognized command-line option '-m64'
------
Dockerfile.arm64:34
--------------------
  33 |     WORKDIR /go/build/nakama
  34 | >>> RUN git checkout --quiet "$commit" && \
  35 | >>>     go build -o /go/build-out/nakama -trimpath -mod=vendor -gcflags "-trimpath $PWD" -asmflags "-trimpath $PWD" -ldflags "-s -w -X main.version=$version -X main.commitID=$commit"
  36 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c git checkout --quiet \"$commit\" &&     go build -o /go/build-out/nakama -trimpath -mod=vendor -gcflags \"-trimpath $PWD\" -asmflags \"-trimpath $PWD\" -ldflags \"-s -w -X main.version=$version -X main.commitID=$commit\"" did not complete successfully: exit code: 1

```

## Context

- [ ] Unity
- [ ] Unreal
- [ ] Other

## Your Environment

- Nakama: X.X.X
- Database: X.X.X
- Environment name and version:
- Operating System and version:

**Comment by zyro** [comment]
Fixed in ac9226ddc59d7efaf9e5bfa1c3cfcae771e866eb. Note it is usually not recommended to build from source or build your own docker images, use existing binaries and images where possible.

**Comment by xinatcg** [comment]
> Fixed in [ac9226d](https://github.com/heroiclabs/nakama/commit/ac9226ddc59d7efaf9e5bfa1c3cfcae771e866eb). Note it is usually not recommended to build from source or build your own docker images, use existing binaries and images where possible.

But there is not arm64 official docker image right?
